src/app/downloads/downloads.routing.ts

```diff
@@ -3,26 +3,25 @@ import { Routes, RouterModule } from '@angular/router';
 import { AuthGuard, DeactivateGuard } from 'ngx-prx-styleguide';
 
 import { DownloadsComponent } from './downloads.component';
-import { NavMenuComponent } from '../shared/nav';
 
 export const downloadsRoutes: Routes = [
   {
-    path: ':seriesId',
+    path: ':podcastId',
     pathMatch: 'full',
-    redirectTo: ':seriesId/downloads/podcast/daily'
+    redirectTo: ':podcastId/reach/podcast/daily'
```

Comment:
should this url be `:seriesId/reach/downloads` per #291, will decide later, but either way currently was not consistent with some of these redirect paths still on `downloads`